frontend/src/index.js

```diff
@@ -26,7 +26,7 @@ ReactDOM.render(
         )} />
         <Route exact path="/Streaming" 
         render={(props) => (
-            <StreamingTablePage {...props} dataSource={DeviceApi} />
+            <StreamingTablePage {...props} dataSource={DeviceApi} streamSource={StreamApi} />
         )}  />
```

Comment:
that's because you're at the homepage, if you click on Streaming, you'll be brought the streaming page where you can see Kouki's work

Reply:
I imagined that was the case, however in that case the breadcrumbs are misleading and should be removed.

Reply:
But then there's no immediate way to navigate to the different pages, without adding a new button or typing it in.

Reply:
I see your point. I think having buttons or typing them in makes more sense, but then again that's subject to debate. Since this page will be revamped in the future, I guess it's alright for it to stay as is, especially since it's not in the scope of this PR.

Reply:
@sharon-ho currently all page navigation is very.. let's say patchworked? i'm assuming we're going to have a big refactoring task in the future to handle it :joy: (revisions for workflows are coming up i believe?)

Reply:
I think we can leave this as is, it's a temporary replacement for a homepage that we'll have to make in a later PR.